ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml

```diff
@@ -86,37 +114,45 @@ Item {
         Row {
             id: collectibleImageDetails
 
-            readonly property real visibleImageHeight: (collectibleimage.visible ? collectibleimage.height : privilegedCollectibleImage.height)
-            readonly property real visibleImageWidth: (collectibleimage.visible ? collectibleimage.width : privilegedCollectibleImage.width)
+            readonly property real visibleImageHeight: artwork.height
+            readonly property real visibleImageWidth: artwork.width
 
             height: collectibleImageDetails.visibleImageHeight
             width: parent.width
             spacing: 24
 
-            // Special artwork representation for community `Owner and Master Token` token types:
-            PrivilegedTokenArtworkPanel {
-                id: privilegedCollectibleImage
-
-                visible: root.isCommunityCollectible && (root.isOwnerTokenType || root.isTMasterTokenType)
-                size: PrivilegedTokenArtworkPanel.Size.Large
-                artwork: collectible.imageUrl
-                color: !!collectible && root.isCommunityCollectible? collectible.communityColor : "transparent"
-                isOwner: root.isOwnerTokenType
-            }
-
-            StatusRoundedMedia {
-                id: collectibleimage
-
-                visible: !privilegedCollectibleImage.visible
-                width: 248
-                height: width
-                radius: Style.current.radius
-                color: collectible.backgroundColor
-                border.color: Theme.palette.directColor8
-                border.width: 1
-                mediaUrl: collectible.mediaUrl ?? ""
-                mediaType: collectible.mediaType ?? ""
-                fallbackImageUrl: collectible.imageUrl
+            ColumnLayout {
+                id: artwork
+                width: 248 + (repeater.count*d.imageStackSpacing)
+                spacing: 0
+                Repeater {
+                    id: repeater
+                    width: parent.width
+                    model: 3//d.balanceAggregator.value < 2 ? 1 : d.balanceAggregator.value === 2 ? 2 : 3
```

Comment:
not sure if the designs changed, the commented version sounds more logical to me

Reply:
also, would LOVE to see what that looks like with an animated image/video :D

Reply:
> not sure if the designs changed, the commented version sounds more logical to me

It also matches the design: 
<img src="https://github.com/status-im/status-desktop/assets/47554641/bdbe3e4e-4c95-4458-8f65-b1771ed2bf86" height = 200/>

However, at first glance it seems this will also do the job: `Math.min(3, d.balanceAggregator.value)`

Reply:
> also, would LOVE to see what that looks like with an animated image/video :D

@dlipicar wondering how I can create an ERC1155 with animated video/image?

Reply:
@Khushboo-dev-cpp add this watched address https://opensea.io/SonnyF